### ISSUE TITLE
Nest Developers Revised Site

### ISSUE THREAD

**Issue by DeaFroggatt** [zpeters/nerdnest#15]
Followed procedures to create Nest Developers Account on the nerdnest wiki..  Although the Product Details and steps that now appear at https://developers.nest.com do not appear as indicated on the page created by Zach Peters, Feb 2, 2017.
Subsequently I had difficulty completing the step "Registering nerdnest with your account".  The Authorization URL took me to a page unrecognized page.
Is there a revised version to these instructions that match the current Nest Developer page??

**Comment by zpeters** [comment]
I updated the descriptions with some more screenshots.  The steps didn't change very much but please go through the wiki and give it a try again.  If you get stuck please let me know and i'll be happy to try and help you out.

-zach

**Comment by DeaFroggatt** [comment]
Thank you Zack.  I went trough and de-installed the device handle and smart things for My-Q and then re-installed fro your revised directions.  All went well and now I can control my  garage door with SmartThings app both with the app and also with the separate buttons I added as devices.

The only issue, may not be related at all to your GitHub Repository.  When I go to the ActionTiles program to add a Tile  and select “Things”, the MY-Q or buttons do not appear as an option to add.

I believe Imay need to see what support I can get from ActionTiles.

Thank you Sincerely.

Here are the screen shots from my SmartThings IDE.
